<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowClass</class>
 <widget class="QMainWindow" name="MainWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>660</width>
    <height>402</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DAC Analyzer</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="margin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QStackedWidget" name="stackedWidget">
      <widget class="QWidget" name="page">
       <widget class="Line" name="line">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>16</width>
          <height>171</height>
         </rect>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_2"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>660</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionFileImport"/>
    <addaction name="actionFileSave"/>
    <addaction name="actionE_xit"/>
   </widget>
   <widget class="QMenu" name="menu_View">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="action_Power_Monitoring"/>
   </widget>
   <widget class="QMenu" name="menu_Settings">
    <property name="title">
     <string>&amp;Settings</string>
    </property>
    <addaction name="action_Clock"/>
    <addaction name="actionDacType"/>
    <addaction name="actionClockFrequency"/>
    <addaction name="action_GPIB"/>
   </widget>
   <widget class="QMenu" name="menuInterface_Control">
    <property name="title">
     <string>Interface &amp;Control</string>
    </property>
    <addaction name="action_Reset_GPIO_SPI"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="action_Help_Topics"/>
    <addaction name="actionAboutCETC58DACAnalyzer"/>
   </widget>
   <widget class="QMenu" name="menuWindow">
    <property name="title">
     <string>&amp;Window</string>
    </property>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_View"/>
   <addaction name="menu_Settings"/>
   <addaction name="menuInterface_Control"/>
   <addaction name="menuWindow"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="dockWidgetConsole">
   <property name="minimumSize">
    <size>
     <width>244</width>
     <height>186</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>244</width>
     <height>524287</height>
    </size>
   </property>
   <property name="features">
    <set>QDockWidget::AllDockWidgetFeatures</set>
   </property>
   <property name="allowedAreas">
    <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>控制台</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>DAC设置</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QDacTypeWidget" name="dacTypeWidget" native="true"/>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>频率设置</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QClockFreqWidget" name="clockFreqWidget" native="true"/>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="title">
        <string>测试窗</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="QTestWidget" name="testWidget" native="true"/>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidgetPowerMonitor">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="floating">
    <bool>true</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::NoDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>电源监控</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <layout class="QGridLayout" name="gridLayout_5">
     <item row="0" column="0">
      <widget class="QPowerMonitor" name="widget" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionFileImport"/>
   <addaction name="actionFileSave"/>
  </widget>
  <action name="actionFileImport">
   <property name="icon">
    <iconset resource="zebra-tester.qrc">
     <normaloff>:/images/images/open.png</normaloff>:/images/images/open.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Import</string>
   </property>
  </action>
  <action name="actionFileSave">
   <property name="icon">
    <iconset resource="zebra-tester.qrc">
     <normaloff>:/images/images/save.png</normaloff>:/images/images/save.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Save</string>
   </property>
  </action>
  <action name="actionE_xit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
  </action>
  <action name="action_Power_Monitoring">
   <property name="text">
    <string>&amp;Power Monitoring</string>
   </property>
  </action>
  <action name="action_Clock">
   <property name="text">
    <string>&amp;Clock Port</string>
   </property>
  </action>
  <action name="actionDacType">
   <property name="text">
    <string>&amp;DAC Type</string>
   </property>
  </action>
  <action name="actionClockFrequency">
   <property name="text">
    <string>Clock &amp;Frequency</string>
   </property>
  </action>
  <action name="action_GPIB">
   <property name="text">
    <string>&amp;GPIB</string>
   </property>
  </action>
  <action name="action_Reset_GPIO_SPI">
   <property name="text">
    <string>&amp;Reset/GPIO/SPI</string>
   </property>
  </action>
  <action name="action_Help_Topics">
   <property name="text">
    <string>&amp;Help Topics</string>
   </property>
  </action>
  <action name="actionAboutCETC58DACAnalyzer">
   <property name="text">
    <string>&amp;About CETC58 DAC Analyzer</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QDacTypeWidget</class>
   <extends>QWidget</extends>
   <header>qdactypewidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QClockFreqWidget</class>
   <extends>QWidget</extends>
   <header>qclockfreqwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QTestWidget</class>
   <extends>QWidget</extends>
   <header>qtestwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QPowerMonitor</class>
   <extends>QWidget</extends>
   <header>qpowermonitor.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="zebra-tester.qrc"/>
 </resources>
 <connections/>
</ui>
